cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/EquipmentExport.java

```diff
@@ -802,10 +802,12 @@ private static String getPhaseTapChangerRegulationMode(PhaseTapChanger ptc) {
 
     private static int getPhaseTapChangerNeutralStep(PhaseTapChanger ptc) {
         int neutralStep = ptc.getLowTapPosition();
-        while (ptc.getStep(neutralStep).getAlpha() != 0.0) {
-            neutralStep++;
-            if (neutralStep > ptc.getHighTapPosition()) {
-                return ptc.getHighTapPosition();
+        double minAlpha = Math.abs(ptc.getStep(neutralStep).getAlpha());
+        for (Map.Entry<Integer, PhaseTapChangerStep> step : ptc.getAllSteps().entrySet()) {
+            double tempAlpha = Math.abs(1 - step.getValue().getAlpha());
```

Comment:
I think we should choose the step with `Alpha` closest to `0`, not the closest to `1`.